Framework/Algorithms/src/ReflectometryReductionOne.cpp

```diff
@@ -612,7 +645,59 @@ void ReflectometryReductionOne::exec() {
   }
 
   IvsQ = this->toIvsQ(IvsLam, correctDetectorPositions, theta, isPointDetector);
-
+  double momentumTransferMinimum = getProperty("MomentumTransferMinimum");
```

Comment:
This `exec` method has become far too large. I suggest creating methods (preferably non-member) for each sub-operation such as `Scale` like [here](https://github.com/mantidproject/mantid/blob/15672_Generalise_ReflTableViewPresenter/Framework/Algorithms/src/ReflectometryReductionOne.cpp#L691-L699)